src/index.scss

```diff
@@ -1,24 +1,31 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900;1000&display=swap');
 
-$blue: #465491;
+$dark-blue: #1B315E;
```

Comment:
Just a thought: one alternate way to do SASS variables is to use `primary-color`, `secondary-color`, `tint-color`, `background-color` as vars -- I personally like what you've done here more than that!